pkg/file/finder_test.go

```diff
@@ -115,7 +116,7 @@ func TestGetGroups(t *testing.T) {
 	excludedFiles := []string{"testdata/go/go.mod", "testdata/misc/requirements.txt"}
 	const nbrOfGroups = 2
 
-	fileGroups, err := finder.GetGroups(path, exclusions, false)
+	fileGroups, err := finder.GetGroups(path, exclusions, false, 0)
```

Comment:
I think you should add consts for all these different levels of strictness so that they are easily editable in the feature